affordance_primitives/include/affordance_primitives/screw_conversions.hpp

```diff
@@ -124,6 +124,35 @@ inline Twist vectorToTwist(const Eigen::VectorXd& vec)
   return output;
 }
 
+/**
+ * @brief Converts a wrench message to a 6x1 vector with [torque ; force]
+ */
+inline Eigen::VectorXd wrenchToVector(const Wrench& wrench)
```

Comment:
Just curious. It seems all the functions in this header are inline functions. While I understand the need to avoid function call overhead (for sufficiently small functions), are we not worried about increased size of the binary executable, thus degrading performance (thrashing in memory)?

Reply:
I've been thinking about this too, I added a note in my TODO's to look at this in a future PR (for all the functions in this header) more closely. I suggest we leave as is for this one